src/components/look-controls.js

```diff
@@ -20,6 +20,15 @@ module.exports.Component = registerComponent('look-controls', {
     this.setupMouseControls();
     this.setupHMDControls();
     this.bindMethods();
+
+    // Enable grab cursor class on canvas.
+    var sceneEl = this.el.sceneEl;
```

Comment:
Sorry, this code convention reduces my creativity and energy because it is such a drag on maintenance and readability. I've adopted most styles so far, but I wouldn't make a big fuss if I didn't think it really impacted my ability to quickly make changes.

Have you checked out #1679? I made arguments that it drastically **increases** cognitive load because you have to load more variables into your head for every function you read and write, and deduce which variables are used where. 

- `Is this variable already defined?` - Linter and text editors catch that. 
- `Is it naming the same thing I want?` - I am not sure what this means.
- `Where is it?` - I am not sure what that means either. I propose variables are declared where you use them, so they will be right next to where you are reading.